tpm/tpm.go

```diff
@@ -929,6 +929,30 @@ func NVReadValue(rw io.ReadWriter, index, offset, len uint32, ownAuth digest) ([
 	return data, nil
 }
 
+func NVWriteValue(rw io.ReadWriter, index, offset uint32, data []byte, ownAuth digest) ([]byte, error) {
```

Comment:
It looks like this function always returns nil in its first return value. So, please return only `error`

Reply:
Whoopsi